spidermon/contrib/scrapy/monitors/base.py

```diff
@@ -1,18 +1,93 @@
+import operator
+import logging
+
 from spidermon import Monitor
 from spidermon.exceptions import NotConfigured
 
 from ...monitors.mixins.spider import SpiderMonitorMixin
 
+logger = logging.getLogger(__name__)
+
 
 class BaseScrapyMonitor(Monitor, SpiderMonitorMixin):
     longMessage = False
+    ops = {
+        ">": operator.gt,
+        ">=": operator.ge,
+        "<": operator.lt,
+        "<=": operator.le,
+        "==": operator.eq,
+        "!=": operator.ne,
+    }
 
     @property
     def monitor_description(self):
         if self.__class__.__doc__:
             return self.__class__.__doc__.split("\n")[0]
         return super().monitor_description
 
+    def run(self, result):
+        if self.check_if_skip_rule_met():
+            logger.info(f"Skipping {self.name} monitor")
+            return
+
+        return super().run(result)
+
+    """ 
+    The montior inherited by BaseScrapyMonitor can be skipped based on
```

Comment:
“The montior inherited by BaseScrapyMonitor” is `Monitor`. Did you mean the following?

```suggestion
    A monitor that inherits BaseScrapyMonitor can be skipped based on
```